bazel/deps.bzl

```diff
@@ -272,8 +272,8 @@ def stratum_deps():
                 "sudo apt-get install -y --no-install-recommends ./linux-headers-4.19.0-12-2-amd64.deb",
                 "rm ./linux-headers-4.19.0-12-2-common.deb ./linux-headers-4.19.0-12-2-amd64.deb",
                 "sudo mkdir -p /usr/src/linux-headers-4.19.0-12-2-merged",
-                "sudo rsync -ahPL /usr/src/linux-headers-4.19.0-12-2-common/ /usr/src/linux-headers-4.19.0-12-2-merged",
                 "sudo rsync -ahPL /usr/src/linux-headers-4.19.0-12-2-amd64/ /usr/src/linux-headers-4.19.0-12-2-merged",
+                "sudo rsync -ahPL /usr/src/linux-headers-4.19.0-12-2-common/ /usr/src/linux-headers-4.19.0-12-2-merged",
```

Comment:
Copying this last does not matter for BCM, but for consistencies sake we fix this here too.
Later we can explore just downloading the merged tar directly.